src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -443,45 +470,46 @@ public C(string a, string b, string c)
     {{
         if (string.IsNullOrEmpty(a))
         {{
-            throw new ArgumentException($""{string.Format(FeaturesResources._0_cannot_be_null_or_empty, "{nameof(a)}")}"", nameof(a));
+            throw new ArgumentException($""{string.Format(FeaturesResources._0_cannot_be_null_or_empty, "{nameof(a)}").Replace("\"", "\\\"")}"", nameof(a));
         }}
 
         if (string.IsNullOrEmpty(b))
         {{
-            throw new ArgumentException($""{string.Format(FeaturesResources._0_cannot_be_null_or_empty, "{nameof(b)}")}"", nameof(b));
+            throw new ArgumentException($""{string.Format(FeaturesResources._0_cannot_be_null_or_empty, "{nameof(b)}").Replace("\"", "\\\"")}"", nameof(b));
         }}
 
         if (string.IsNullOrEmpty(c))
         {{
-            throw new ArgumentException($""{string.Format(FeaturesResources._0_cannot_be_null_or_empty, "{nameof(c)}")}"", nameof(c));
+            throw new ArgumentException($""{string.Format(FeaturesResources._0_cannot_be_null_or_empty, "{nameof(c)}").Replace("\"", "\\\"")}"", nameof(c));
         }}
     }}
-}}", index: 3);
+}}",
+                CodeActionIndex = 3,
+                CodeActionEquivalenceKey = FeaturesResources.Add_null_checks_for_all_parameters
```

Comment:
💡 It's better to use non-localized names for the equivalence keys. `nameof` makes this easy.

```suggestion
                CodeActionEquivalenceKey = nameof(FeaturesResources.Add_null_checks_for_all_parameters),
```